nixos/doc/manual/release-notes/rl-2411.section.md

```diff
@@ -572,41 +580,34 @@
 
 - `services.timesyncd.servers` now defaults to `null`, allowing systemd-timesyncd to use NTP servers advertised by DHCP.
 
 - `services.timesyncd.fallbackServers` was added and defaults to `networking.timeServers`.
 
-- Cinnamon has been updated to 6.2, please check [upstream announcement](https://www.linuxmint.com/rel_wilma_whatsnew.php) for more details.
-  Following Mint 22 defaults, the Cinnamon module no longer ships geary and hexchat by default.
+- Cinnamon has been updated to 6.2. Please check [upstream announcement](https://www.linuxmint.com/rel_wilma_whatsnew.php) for more details.
+  Following Mint 22 defaults, the Cinnamon module no longer ships `geary` and `hexchat` by default.
 
 - `zfs.latestCompatibleLinuxPackages` is deprecated and is now pointing at the default kernel. If using the stable LTS kernel (default `linuxPackages` is not possible then you must explicitly pin a specific kernel release. For example, `boot.kernelPackages = pkgs.linuxPackages_6_6`. Please be aware that non-LTS kernels are likely to go EOL before ZFS supports the latest supported non-LTS release, requiring manual intervention.
 
 - The `shadowstack` hardening flag has been added, though disabled by default.
 
-- `xxd` is now provided by the `tinyxxd` package, rather than `vim.xxd`, to reduce closure size and vulnerability impact. Since it has the same options and semantics as Vim's `xxd` utility, there is no user impact. Vim's `xxd` remains available as the `vim.xxd` package.
-
-- `prometheus-openldap-exporter` was removed since it was unmaintained upstream and had no nixpkgs maintainers.
+- `xxd` is now provided by the `tinyxxd` package rather than `vim.xxd` to reduce closure size and vulnerability impact. Since it has the same options and semantics as Vim's `xxd` utility, there is no user impact. Vim's `xxd` remains available as the `vim.xxd` package.
 
-- `restic` module now has an option for inhibiting system sleep while backups are running, defaulting to off (not inhibiting sleep), available as [`services.restic.backups.<name>.inhibitsSleep`](#opt-services.restic.backups._name_.inhibitsSleep).
+- `restic` module now has an option for inhibiting system sleep while backups are running, defaulting to off (not inhibiting sleep). Available as [`services.restic.backups.<name>.inhibitsSleep`](#opt-services.restic.backups._name_.inhibitsSleep).
 
-- The arguments from [](#opt-services.postgresql.initdbArgs) now get shell-escaped.
+- The arguments from [services.postgresql.initdbArgs](#opt-services.postgresql.initdbArgs) now get shell-escaped.
```

Comment:
Doesn’t the manual generator already handle including the name here? If not, shouldn’t this at least be in backticks?

Reply:
It does! TIL